src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/ChooseConnection.java

```diff
@@ -32,9 +32,9 @@
 	private static final long serialVersionUID = 4773092967249064165L;
 
 	private final ToggleButtonIcon bConnect = new ToggleButtonIcon(
-			asList("ButtonConnect", "ButtonDisconnect"),
-			asList("/images/connect.png", "/images/disconnect.png"),
-			asList(Color.GREEN, Color.RED));
+			new ToggleButtonIcon.Item("ButtonConnect", "/images/connect.png", Color.GREEN),
+			new ToggleButtonIcon.Item("ButtonDisconnect", "/images/disconnect.png", Color.RED)
+		);
```

Comment:
nice.

Reply:
I can only say the same, @coliss86  did a good job. :)